gis4wrf/plugin/ui/tab_home.py

```diff
@@ -1,16 +1,51 @@
 # GIS4WRF (https://doi.org/10.5281/zenodo.1288569)
 # Copyright (c) 2018 D. Meyer and M. Riechert. Licensed under MIT.
 
-from PyQt5.QtWidgets import QWidget
 from pathlib import Path
 
-from gis4wrf.plugin.ui.helpers import create_browser_layout
 
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QWidget, QVBoxLayout, QLabel
+from PyQt5.QtGui import QPixmap
+from gis4wrf.plugin.ui.helpers import FormattedLabel
+from gis4wrf.plugin.constants import GIS4WRF_LOGO_PATH
 
 class HomeTab(QWidget):
     """Class for creating the Home tab"""
-
     def __init__(self) -> None:
         super().__init__()
-        page = Path('tab_home', 'index.html')
-        self.setLayout(create_browser_layout(page))
+        vbox = QVBoxLayout()
+        title = """
+                    <html>
+                        <h1>Welcome to GIS4WRF</h1>
+                        <br>
+                    </html>
+                """
+        text = """
+                    <html>
+                        <font size="4">
+                        <br>
+                        <p>The GIS4WRF documentation and tutorials have been moved online at: <a href="https://gis4wrf.github.io">https://gis4wrf.github.io</a></p>
+                        <br>
+                        <p>We are delighted to announce that we can now provide MPI-enabled WPS-V4 and WRF-V4 pre-built binaries for Windows, macOS and Linux through WRF-CMake. 😊</p>
+                        <p>If you have not done so already, make sure to download the latest V4 releases for your system.
+                            For more info see: <a href="https://gis4wrf.github.io/configuration">https://gis4wrf.github.io/configuration</a></p>
```

Comment:
Is it normal that I can't click any of the links?

Reply:
Fixed it, had to do with the FormattedLabel class which wasn't used in the other cases, dead code, and this breaks hyperlinks for some reason.